docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -333,3 +333,41 @@ Mouse event | Touch event
 `mousemove` (when hovering or dragging) | `touchmove` (dragging only)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Interaction Requirements
+
+To be properly targeted by TestCafe, elements have to satisfy several conditions:
+
+* an element has to be within the page [`body`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or in an [`<iframe>`](#work-with-iframes).  
+
+* an element has to be visible- neither of its properties should be set to:
+
+    Property | Value
+    -------- | --------
+    `display`  | `none`
+    `visibility` | `hidden`, `collapse`
+    `width`    | `0`
+    `height`   | `0`
```

Comment:
There is a thing with human perception - we are likely to ignore the text and focus on images, code examples, tables. So, many people would skip "neither" and look at the values.

The same problem occurs when we show an example that shouldn't work (to show how not to do). In this instance, we would have to indicate this visually (like with a red background). Otherwise, many will think it's a good example.

There are several ways to address this, including inversion (it brings a little complication because we should take into account the default values):

```
an element has to be visible - its properties should be set as follows:

    Property | Value
     -------- | --------
     `display`  | *not* set to `none`
     `visibility` | set to `visible` (the default value)
     `width`    | auto-calculated as or set to > `0`
     `height`   | auto-calculated as or set to > `0`
```

Reply:
visibility can be **visible**|hidden|collapse|**initial**|**inherit**; we're looking for the highlighted ones here.
can we write:

```
Property | Value
     -------- | --------
     `visibility` | *not* set to `hidden` or `collapse`
```